packages/time-series-chart/static/index.html

```diff
@@ -0,0 +1,30 @@
+<!--
+  ~ Copyright 2020 Red Hat, Inc. and/or its affiliates.
```

Comment:
Could you please update to "2021" and also add license headers to the other files that we can add comment (see my others comments)